### ISSUE TITLE
pip repository @z43 for internal packages

### ISSUE THREAD

**Issue by pcrespov** [ITISFoundation/osparc-simcore#590]
- [x] Deploy pip repo in-house (see docker preconfigured)
- This repo will contain released versions of internal python packages (e.g. Murphy or rec5g)
- [x] Any z43 gitlab user can ``pip install`` these packages within z43 network
- CI tools creates and pushed release versions of these packages

Access and maintenance (e.g. backup) should be analogous to git repos

**Comment by pcrespov** [comment]
```console
$ devpi login username 
  WARN: devpi-client-4.4.0 got an unversioned reply, assuming API-VERSION 2 (as implemented by devpi-server-2.0)
POST https://devpi.speag.com/+login
500 Internal Server Error: Internal Server Error

The server encountered an unexpected internal server error

(generated by waitress)
```

The log of the devpi server logs an error in the db tables:
![image](https://user-images.githubusercontent.com/32402063/57723816-bde4db00-7689-11e9-81ab-f935fe8617cb.png)

**Comment by pcrespov** [comment]
@oetiker did you have a chance to look at this? ;-)
